control_panel_api/serializers.py

```diff
@@ -46,11 +46,30 @@ class Meta:
     def update(self, instance, validated_data):
         if instance.app != validated_data['app']:
             raise serializers.ValidationError(
-                "app can't change, create a new record"
+                "App is not editable. Create a new record."
             )
         if instance.s3bucket != validated_data['s3bucket']:
             raise serializers.ValidationError(
-                "s3bucket can't change, create a new record"
+                "S3Bucket is not editable. Create a new record."
+            )
+
+        return super().update(instance, validated_data)
+
+
+class UserS3BucketSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = UserS3Bucket
+        fields = ('id', 'url', 'user', 's3bucket', 'access_level')
```

Comment:
@jmoz just realised, shouldn't we expose `is_admin` as well here?

Reply:
Yes, well spotted, thanks.